jsx/Form.js

```diff
@@ -21,41 +23,14 @@ NumericElement, FileElement, StaticElement, LinkElement, ButtonElement, LorisEle
  * Note that if both are passed `this.props.formElements` is displayed first.
  *
  */
-var FormElement = React.createClass({
-
-  propTypes: {
-    name: React.PropTypes.string.isRequired,
-    id: React.PropTypes.string,
-    method: React.PropTypes.oneOf(['POST', 'GET']),
-    action: React.PropTypes.string,
-    class: React.PropTypes.string,
-    columns: React.PropTypes.number,
-    formElements: React.PropTypes.shape({
-      elementName: React.PropTypes.shape({
-        name: React.PropTypes.string,
-        type: React.PropTypes.string
-      })
-    }),
-    onSubmit: React.PropTypes.func,
-    onUserInput: React.PropTypes.func
-  },
+class FormElement extends React.Component {
+  constructor() {
+    super();
```

Comment:
should be super(props) and constructor(props)

Reply:
@xlecours @davidblader what are your thoughts on this?

Should we keep `props` as an argument in all constructors?

Reply:
According to my reading, `super(props)` is useful only when we need to use the `this.props` in the constructor. Since we don't, there is no need to have `super(props)` hence `constructor(props)`.

Reply:
My opinion is that we should keep super(props). If the class's parent takes a constructor, it should be passed (even if we don't use it.)